### ISSUE TITLE
How can I install this?

### ISSUE THREAD

**Issue by walidmahade** [FifthRooter/OpenShotX#1]
Any guide would be helpful for new users about how to install this!

**Comment by risenowrise** [comment]
same.  there seems to be no requirements.txt   for pip?

**Comment by FifthRooter** [comment]
guys i'll have to disappoint you but i've made very little progress on this project since its inception. been busy with other things, but it's definitely in my backlog because, still, the existing screen capture solutions are severely lacking. really sorry!

**Comment by undefinedcode0** [comment]
clone the repo, get rust, cd into the directory, and run cargo build. theres no python code in this (i believe)

**Comment by FifthRooter** [comment]
yes indeed, there's no python present, only rust. please anyone with any time on their hands clone the repo and see how far you can get. i'm bringing this project back online so will see more active development here.

**Comment by undefinedcode0** [comment]
the repo can only be built as a library, so you'll have to code your own main.rs for this to actually work.

**Comment by FifthRooter** [comment]
guys we're back in business. shit's very raw but stuff is beginning to work. oh, how i missed quick access OCR!
only cli commands at the moment, but i'm trying to burn through all the important features before cleaning it up and throwing in hotkeys etc.

**Comment by TheKelvinPerez** [comment]
I love cleanshot x use it daily on macos and i've been slowly transitioning to linux and i use flameshot but its not even close to the DX of cleanshot x i would love to contribute to this repo because i was planning on building my own version i'll check out what you have this week

Do you have a public roadmap for this and the features you plan on implimenting?
